docs/en/observability/synthetics-configuration.asciidoc

```diff
@@ -224,9 +224,24 @@ By default, monitors are automatically retested if the monitor goes from "up" to
 If the result of the retest is also "down", an error will be created, and if configured, an alert sent.
 Then the monitor will resume running according to the defined schedule.
 Using `retestOnFailure` can reduce noise related to transient problems.
+
+`fields` (`object`)::
+A list of key-value pairs that will be sent with each monitor event.
+This is useful for adding custom metadata to your monitor.
```

Comment:
additional info that is important is that we should mention that data from fields is always appended under labels in es documents. we also display those labels in UI in monitor details panel below tags 
<img width="1727" alt="image" src="https://github.com/user-attachments/assets/38301aae-ba44-4cba-be24-d3eb078367a2">